alpha/notify-worker/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM node:6.5.0
+
+RUN mkdir -p /usr/src/app
+WORKDIR /usr/src/app
+
+COPY package.json /usr/src/app/
+COPY . /usr/src/app
+
+CMD npm install && node start.js
```

Comment:
npm install should be moved to image build so dependencies are baked into image and not resolved at run time (only resolving deps at run time for development work)